corehq/apps/userreports/reports/view.py

```diff
@@ -321,3 +337,26 @@ def excel_response(self):
         file = StringIO()
         export_from_tables(self.export_table, file, Format.XLS_2007)
         return file
+
+
+class CustomConfigurableReportDispatcher(ReportDispatcher):
+    slug = prefix = 'custom_configurable'
+    map_name = 'CUSTOM_UCR'
+
+    def dispatch(self, request, *args, **kwargs):
+        domain = kwargs.get('domain')
+        report_config_id = kwargs.get('report_config_id')
+        class_path = CustomReportConfiguration.report_class_by_domain_and_id(
+            domain, report_config_id
+        )
+        report_class = to_function(class_path)
```

Comment:
@czue We had talked about avoiding specifying python paths due to the complications of refactoring, but my attempts to avoid this descended into circular import hell, so I went with this approach.